server/models.py

```diff
@@ -404,6 +404,10 @@ class Batch(BaseModel):
         order_by="BatchResultTallySheet.created_at",
     )
 
+    last_edited_by_user_type = Column(String(200))
+    last_edited_by_user_key = Column(String(200))
+    last_edited_by_support_user_email = Column(String(200))
```

Comment:
I considered a number of possible schema, including foreign keys into the users / tally entry users tables, but ultimately settled on these three session fields as a simple way to identify users of various types